samples/SqlExtensionSamples/local.settings.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
We shouldn't be checking this in - especially for samples that people might be looking at. Something like this should probably be generated by the pipeline itself.

Reply:
Wouldn't we want to include this for people to look at? They will need these settings to run the sample functions (especially the SqlConnectionString which is specific to our samples)

Reply:
I was basing it off of the guidance from the AF docs about never checking this in. But I guess we have the .gitignore set up properly so maybe it's not the worst idea...seems like other sample guides have it e.g. https://github.com/yokawasa/azure-functions-python-samples/tree/master/v2functions...

I guess as long as it's empty like this it's fine